### PR TITLE
Added delete reports endpoint along with a helper method. Also swappe…

### PR DESCRIPTION
**Pull request by stevieing**
…d data files around a bit so they can be used in tests.

Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: lighthouse/blueprints/reports.py [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
 
-from flask import Blueprint
+from flask import Blueprint, request
 from flask_cors import CORS  # type: ignore
 
-from lighthouse.helpers.reports import get_reports_details
+from lighthouse.helpers.reports import get_reports_details, delete_reports
 from lighthouse.jobs.reports import create_report
 
+import json
```
Comment (harrietc52) [inline lighthouse/blueprints/reports.py:11]: Is this being used

File: lighthouse/blueprints/reports.py [hunk 2]
```diff
@@ -34,3 +36,24 @@ def create_report_endpoint():
...
+    from the reports path. 
+    This endpoint should be json and the body should be in
+    the format {"data":"filenames":["file1.xlsx","file2.xlsx", ...]}
+    This is a POST request but is a destructive action but this does not need to be
+    delete as it is not a REST resource.
+    Arguments:
+        None
+    Returns:
+        {}, HTTPStatus
+    """
```
Comment (harrietc52) [inline lighthouse/blueprints/reports.py:53]: Nice comment 👍

File: lighthouse/helpers/reports.py [hunk 3]
```diff
@@ -100,7 +100,24 @@ def get_new_report_name_and_path() -> Tuple[str, pathlib.PurePath]:
...
+        if (coordinate and isinstance(coordinate, str))
+        else coordinate
+    )
+
+
+def delete_reports(filenames):
+    """delete reports from the standard reports folder if they exist.
+
+    Returns:
+        Nothing.
```
Comment (harrietc52) [inline lighthouse/helpers/reports.py:118]: Maybe the successfully deleted filenames could be returned from this method, and the request then displayed in the UI success alert

File: lighthouse/blueprints/reports.py (hunk 1 above)
Comment (stevieing) [reply lighthouse/blueprints/reports.py:11]: Not anymore. I removed it's use as of this afternoon. I will remove it.

File: lighthouse/helpers/reports.py (hunk 3 above)
Comment (stevieing) [reply lighthouse/helpers/reports.py:118]: I did think about that but wondered whether it was worth it? Kind of like saying this is what you are missing nothing you can do about it :(

**Pull request merged**